src/services/projects.service.ts

```diff
@@ -34,50 +34,60 @@ export class ProjectsService {
 	}
 
 	static async build(projectId: string, resources: Record<string, Uint8Array>): Promise<ServiceResponse<string>> {
-		const { error: resourcesError } = await this.setResources(projectId, resources);
-		if (resourcesError) {
-			LoggerService.error(`${namespaces.projectService} - Upload resources`, (resourcesError as Error).message);
+		try {
+			const { error: resourcesError } = await this.setResources(projectId, resources);
+			if (resourcesError) {
+				LoggerService.error(`${namespaces.projectService} - Upload resources`, (resourcesError as Error).message);
```

Comment:
missing context in error, you need the project id here

Reply:
removed